modules/local/omextractor/meta.yml

```
@@ -9,9 +9,13 @@ tools:
         Suite of tools to handle several imaging protocols.
       homepage: https://www.openmicroscopy.org/bio-formats/
       documentation: https://www.openmicroscopy.org/bio-formats/
-      licence: ["GPL-3.0-or-later"]
-      identifier: biotools:multiqc
+      licence: ["GPL-2.0"]
 input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test' ]
   - - image:
```

This is structured wrong. It needs to have `meta` and `image` together as a 2-entry list. The way it's written now makes two 1-entry lists instead which doesn't match the actual input spec in the code. Replacing the first dash with a space should fix it:
```suggestion
    - image:
```